### ISSUE TITLE
Django 1.8 and Translations

### ISSUE THREAD

**Issue by TheRightChoyce** [django-money/django-money#118]
I monkey patched the pull request in #114, however I ran into another issue when translations are getting deferred: In Django 1.8 `translation.get_translation()` has changed to return None when translations are being deferred ([link](https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.translation.get_language)), and in my case, line 105 of models/fields.py 

```
locale = translation.to_locale(translation.get_language())
```

was throwing the error: _AttributeError: 'NoneType' object has no attribute 'find'_ since `translation.get_language()` was returning None with the new functionality. I fixed my local version the following way, however this seems pretty incorrect in the grand scheme of things

```
language = translation.get_language() or settings.LANGUAGE_CODE
locale = translation.to_locale(language)
```

I don't really know the proper way to fix this, so leaving here and hopefully someone with more knowledge of both code-bases can fix = )

**Comment by briankung** [comment]
Would you be able to write a failing test for the translation error and publish your code somewhere?

**Comment by TheRightChoyce** [comment]
It may take me a little while to get to it, but I'll try putting one together as soon as I'm back on the project where I found this.

**Comment by briankung** [comment]
:+1:

**Comment by YAmikep** [comment]
Got the same issue on django 1.8. I did not see this ticket so I actually created [one](https://github.com/django-money/django-money/issues/124) too. Sorry about that.

I used the same fix @TheRightChoyce mention and created a pull request too: https://github.com/django-money/django-money/pull/125

Any change to see this bug fixed soon? Thanks

**Comment by benjaoming** [comment]
Fixed in #125
